layouts/_default/single.html

```diff
@@ -1,3 +1,15 @@
 {{ define "main" }}
-	{{ .Content }}
+<main class="d-flex flex-xl-row flex-column">
+  <aside class="text-muted align-self-start mb-3 px-2 d-xl-none d-block">
+    <h2 class="h6 my-2">On this page</h2>
+    {{ .TableOfContents }}
+  </aside>
```

Comment:
This one is displayed when the viewport is not extra large (because of the `d-xl-none`).